### PR TITLE
7.0 - Upgrade Canada payroll account

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by bwrsandman** [comment]
Rebase this branch after merge of ~~#68~~

**Comment by bwrsandman** [comment]
Needs rebase

File: l10n_ca_hr_payroll/__openerp__.py [hunk 1]
```diff
@@ -50,6 +52,8 @@
         'l10n_ca_hr_payroll_data.xml',
     ],
     'test': [],
-    'demo': [],
+    'demo': [
+        'demo/salary_rule_data.xml',
```
Comment (bwrsandman) [inline l10n_ca_hr_payroll/__openerp__.py]: This file doesn't exist

File: l10n_ca_hr_payroll/__openerp__.py [hunk 2]
```diff
@@ -50,6 +52,8 @@
         'l10n_ca_hr_payroll_data.xml',
```
Comment (bwrsandman) [inline l10n_ca_hr_payroll/__openerp__.py]: Wrong paths

File: l10n_ca_hr_payroll/__openerp__.py (hunk 1 above)
Comment (ghost) [inline l10n_ca_hr_payroll/__openerp__.py]: Fixed.

**Comment by ghost** [comment]
@max3903 you may remove the need fixing label. As far as I know, this module doesn't need fixing. Travis should be green when https://github.com/OCA/l10n-canada/pull/43 is merged.

**Comment by bwrsandman** [comment]
Depends on:
- [ ] Improve Canada Payroll #43 
  - [x] Partner aging OCA/account-invoice-reporting#32

**Pull request closed without merging**